### ISSUE TITLE
aria-allowed-attr: Ensures ARIA attributes are allowed for an element's role

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#4699]
### Describe the issue

Run Accessibility insights extension fast pass, it detected issues on workflow run history view. Please see image below.
![image](https://github.com/Azure/LogicAppsUX/assets/22843713/749a2993-3083-483d-ac09-d507388c46f6)

Failure details:
Title: WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (.react-flow__edge-BUTTON_EDGE.react-flow__edge:nth-child(1))
Tags: Accessibility, WCAG 4.1.2, aria-allowed-attr

Issue: Ensures ARIA attributes are allowed for an element's role (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr)

Target application: IcM - https://ppeportal.microsofticm.com/imp/v5/automation/workflows/AnalyzeTrigger/run/08584876686816696416889735856CU60

Element path: .react-flow__edge-BUTTON_EDGE.react-flow__edge:nth-child(1)

Snippet: <g class="react-flow__edge react-flow__edge-BUTTON_EDGE nopan" data-testid="rf__edge-manual-Initialize_variable" aria-label="Edge from manual to Initialize_variable">

How to fix: 
Fix all of the following:
  aria-label attribute cannot be used on a g with no valid role attribute.

Environment: Microsoft Edge version 124.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Browser: Edge

logic app library version:

"@microsoft/logic-apps-designer": "4.1.1",
"@microsoft/logic-apps-shared": "4.1.1"

**Comment by hartra344** [comment]
This one is due to a dependency(react flow) and is not something we can do anything about but also is a nonfocusable element, so shouldn't matter since a screen reader will never be able to read it.
